src/commcare_cloud/ansible/roles/devops_scripts/tasks/main.yml

```diff
@@ -8,29 +8,6 @@
     mode: "u=rwx,g=rwx,o=r"
   when: "'postgresql' in group_names or 'citusdb' in group_names or 'pg_standby' in group_names"
 
-- name: Copy uptime.sh script to /usr/local/sbin
-  template:
-    src: uptime.sh.j2
-    dest: /usr/local/sbin/uptime.sh
-    owner: ansible
-    group: ansible
-    mode: "u=rwx,g=rwx,o=r"
-  when: "'formplayer' in group_names or 'webworkers' in group_names"
-
-- name: Creates a cron to check the uptime details
-  become: yes
-  cron:
-    name: Run uptime for every minute
```

Comment:
Since we only restart every 90 days, can we consider only running this once per day to match our uptime_limit unit? At the very least, every hour seems sufficient. Since you're calling ``` shutdown -r +1 ``` in uptime.sh it seems like we should avoid running this script every minute.

Reply:
Sure @amerry19 Thankyou for your advice and I'll look into it.